e2e/tests/init_test.go

```diff
@@ -31,12 +33,21 @@ var (
 	accNameToAppMap      = make(map[string]apptypes.Application)
 	accNameToSupplierMap = make(map[string]sharedtypes.Supplier)
 
-	keyRingFlag = "--keyring-backend=test"
+	featuresPathFlag string
```

Comment:
NIT: Let's go with `flagFeaturesPath` to stay consistent with cosmos. https://pkg.go.dev/github.com/cosmos/cosmos-sdk/client/flags